### DISCUSSION TITLE
Cannot find option to add Project (Classic) in a repository

### DISCUSSION THREAD

**Discussion by saquibaijaz** [community/community#37356] [New to GitHub]
Hi
I have the option to add Project (Classic) in a repository but cannot find that option in another repository. Check Out the attached screenshots.
Please Help
![bhanwari](https://user-images.githubusercontent.com/111858763/198041502-fc484ab2-8ab2-4ee3-9d64-85c0a674b729.png)
![sansaar](https://user-images.githubusercontent.com/111858763/198041507-277f2329-530b-4ad9-874b-b5501a254cdc.png)

**Comment by ossanna16** [comment]
Hi there @saquibaijaz and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by queenofcorgis** [comment]
Hi @saquibaijaz - have you checked to make sure Projects (Classic) is enabled in your repository's settings. 

To check: 

Navigate to your repository > Settings > General > Features Section

**Reply by saquibaijaz** [reply]
Hi @queenofcorgis 
Thank you for your reply.
I cannot see the option to enable Project (Classic).
I can only enable Projects, which allows me to create Projects at the Organisation level; Project (Classic) can be used to create projects at the repository level.
Please check the attached screenshot
![Project (Classic)](https://user-images.githubusercontent.com/111858763/199562237-fdcd99b6-1dcf-4a30-9bd4-f4649857703e.png)

**Reply by queenofcorgis** [reply]
Is this repo part of an organization that has turned off this feature? Or an Enterprise account?

**Reply by saquibaijaz** [reply]
Yes, this repo is part of an organisation, but the same organisation have many other repos where the Project (Classic) is enabled. Only in this repo I cannot find the Project (Classic)
Please check the attached screenshot from another repo.
![Project (Classic)](https://user-images.githubusercontent.com/111858763/199978908-12d0e860-b2ef-4363-8e46-58cd34bead90.png)
